e2e/global/cli_test.go

```diff
@@ -0,0 +1,24 @@
+package global
+
+import (
+	"testing"
+
+	"github.com/docker/cli/internal/test/environment"
+	"gotest.tools/icmd"
+	"gotest.tools/skip"
+)
+
+func TestTLSVerify(t *testing.T) {
+	// Remote daemons use TLS and this test is not applicable when TLS is required.
+	skip.If(t, environment.RemoteDaemon())
```

Comment:
Perhaps put the "skip" description as third argument to `skip.If`, then it shows up in the logs as reason why it's skipped